.github/workflows/test.yml

```diff
@@ -14,10 +14,26 @@ jobs:
           - stable
           - beta
           - nightly
-
+    services:
+      postgres:
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_PASSWORD: hilow
+          POSTGRES_USER: hilow
+          POSTGRES_DB: hilow
+    env:
+      DATABASE_URL: postgres://hilow:hilow@localhost/hilow
     steps:
       - uses: actions/checkout@v2
-
+      - name: Run migrations
+        run: |
+          cargo install sqlx-cli --no-default-features --features postgres
+          # Run migrations once they actually exists
+          # cargo sqlx migrate run
+      - name: Check sqlx metadata file
+        run: cargo sqlx prepare --check -- --bin main
```

Comment:
This probably will break in the next PR. If that happens, we can just remove this step. @EstebanBorai

Reply:
nm, it just worked today.